src/react/components/common/imageMap/imageMap.tsx

```diff
@@ -402,6 +438,24 @@ export class ImageMap extends React.Component<IImageMapProps> {
         }
 
         const eventPixel =  this.map.getEventPixel(event.originalEvent);
+        let selectFlag = false;
```

Comment:
Maybe better to just return when a feature is selected, instead of setting and checking a flag. It would prevent cascading and any additional checks seem unnecessary